api/src/main/java/org/openmrs/Location.java

```diff
@@ -792,4 +794,20 @@ public String getAddress15() {
 	public void setAddress15(String address15) {
 		this.address15 = address15;
 	}
+	
+	/**
+	 * @return Returns If a location supports Visits
```

Comment:
```suggestion
	 * @Returns If a location supports Visits
```

Reply:
@sherrif10 `@return` is [a Javadoc tag](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDJGIJB) `@Returns` is not.

Reply:
Thanks @ibacher . Sorry i meant this 
```suggestion
	 * @return If a location supports Visits
```